src/main/java/com/odysseusinc/arachne/datanode/dto/atlas/Pathway.java

```diff
@@ -0,0 +1,155 @@
+package com.odysseusinc.arachne.datanode.dto.atlas;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.odysseusinc.arachne.datanode.dto.serialize.MultiFormatDateDeserializer;
+import java.util.Date;
+import java.util.List;
+
+public class Pathway extends BaseAtlasEntity {
+
+    private Long id;
+    private String name;
+    private List<PathwayCohort> targetCohorts;
+    private List<PathwayCohort> eventCohorts;
+    private Integer combinationWindow;
+    private Integer minCellCount;
+    private Integer maxDepth;
+    private boolean allowRepeats;
+    private AtlasUserDTO createdBy;
+    @JsonDeserialize(using = MultiFormatDateDeserializer.class)
+    private Date createdDate;
+    private AtlasUserDTO modifiedBy;
+    @JsonDeserialize(using = MultiFormatDateDeserializer.class)
+    private Date modifiedDate;
+    private Integer hashCode;
```

Comment:
Just a question which meaning this hachCode has?

Reply:
`hashCode` is computed from analysis design by Atlas. This allows to easily detect are a couple of analyses the same or not.